### PR TITLE
Fix sidebar colors in various themes

### PR DESCRIPTION
**Pull request by flipsimon**
A few new css variables for sidebar colors. Left light theme as it was before. Based on this model I chose the background color of the account section of the sidebar in each theme to be a little lighter than the navigation bar.

image

### CODE REVIEW

File: themes/minimal.scss [hunk 1]
```diff
@@ -14,17 +14,21 @@ $navbarBgLight: $borderColor;
 $outlineColor: #b9b9b9;
```
Comment (flipsimon) [inline themes/minimal.scss:14]: added .scss file ending. was it missing on purpose?

Comment (flipsimon) [inline themes/minimal.scss:14]: added .scss file ending. was it missing on purpose?

Comment (Simon-Laux) [reply themes/minimal.scss:14]: don't know, is there a reason behind it @Jikstra?

**Review by Simon-Laux — APPROVED** [review]
code looks perfect

File: themes/minimal.scss (hunk 1 above)
Comment (Jikstra) [reply themes/minimal.scss:14]: Nope dont think so

**Pull request merged**